cfpb/owning-a-home
# 404 browser tests for form resources + form explainers

Moves the 404 browser tests out of the journey steps and into the navigation steps, the tests are then run against journey, form resources and form explainer pages.

## Additions

- Mortgage closing feature test
- Mortgage estimate feature test

## Removals

- Journey 404 tests - these were moved out of the journey-specific browser tests into the navigation tests

## Changes

- failed URLs in the 404 tests now print individually to the console
- I fixed some URLs that were broken on form explainers
- renamed functions and the tag for this test to simplify
- simplified local link conditional which doesn't include localhost links in the error results when running browser tests locally

## Testing

- check out branch
- run tests:
 -  `behave -k -t=404`
 -  `behave -k -t=journey`

## Review

- @fna 
- @OrlandoSoto 

## Screenshots

### example of console output for a failed link 
![screen shot 2015-08-28 at 12 18 50 pm](https://cloud.githubusercontent.com/assets/702526/9551405/ffeb85a4-4d7e-11e5-9c5e-dd20082ff58f.png)

## Checklist
